.pip-tools.toml

```diff
@@ -0,0 +1,6 @@
+[tool.pip-tools]
+resolver = "backtracking"
+allow-unsafe = true
+strip-extras = true
+quiet = true
+output-file = "docs/docsite/requirements.txt"
```

Comment:
I wouldn't keep it here since it'll be confusing to read the CLI command with an input and no outputs. Besides when adding more constraint files for other envs, it'll be an incorrect value derived from the global config. It's too specific to be global.